modules/delegates.js

```diff
@@ -720,23 +688,27 @@ Delegates.prototype.updateForgingStatus = function(
 /**
  * Gets delegate list based on input function by vote and changes order.
  *
- * @param {number} round
- * @param {function} source - Source function for get delegates
+ * @param {number} blockHeight
  * @param {function} cb - Callback function
  * @param {Object} tx - Database transaction/task object
  * @returns {setImmediateCallback} cb, err, truncated delegate list
  * @todo Add description for the params
  */
-Delegates.prototype.generateDelegateList = function(round, source, cb, tx) {
-	// Set default function for getting delegates
-	source = source || __private.getKeysSortByVote;
+Delegates.prototype.generateDelegateList = function(blockHeight, cb, tx) {
```

Comment:
You haven't renamed this method to `getForgersList` as the PR description states.

Reply:
Yes you are right, we decided to keep it this way after discussing it with @4miners. I updated pr description and I commented on the original issue.